packages/actions/src/ActionHelpers.ts

```diff
@@ -162,6 +165,9 @@ function setVariableValue(variableName: string | Variable, value: VariableValue)
 /**
  * Sets the value of a text variable by its name or variable object.
  *
+ * If the value is set on a ShortTextVariable, it cannot contain any type of
```

Comment:
The contextual data being send is ```"name": "qr", "type": "shortText"```. 

It might be better to mention "shortText" instead?

```suggestion
 * If the value is set on a variable of type shortText, it cannot contain any type of
```

_But even with this change it will create a single line string with "\n" inside of it._

Reply:
wdyt @brapoprod ?